docs/manual/config/index.rst

```diff
@@ -145,6 +145,15 @@ configuration variables that control specific aspects of Qtile's behavior:
       - ``True``
       - Controls whether or not to automatically reconfigure screens when there
         are changes in randr output configuration.
+    * - ``auto_minimize``
+      - ``True``
+      - If things like steam games want to auto-minimize themselves when losing
+        focus, should we respect this or not?
+    * - ``repl_port``
+      - ``41414``
```

Comment:
Is there a reason to bind this on tcp and not just a (different) unix socket?

Reply:
Only because that was the example code I found.